### PR TITLE
Clown Psychedelic Goblin

### PR DESCRIPTION
**Pull request by GermanSanta**
Isn't life just one big joke?
🆑
 - rscadd: Adds psychedelic clown goblins, made from psychedelic clown outfits

### CODE REVIEW

**Comment by GermanSanta** [comment]
GbL_Y53tT6W86pVaDDMmnA

File: config-example/admins.txt [hunk 1]
```diff
@@ -33,6 +33,6 @@ tastyfish - Retired Admin
 uristqwerty - Game Master
 wrongnumber - Game Admin
 mij - Game Admin
-
+germansanta - Game Admin
```
Comment (boy2mantwicethefam) [inline config-example/admins.txt]: based

Comment (GermanSanta) [reply config-example/admins.txt]: would be, if it wasn't merging an example text lmao

**Comment by GermanSanta** [comment]
output

**Comment by PrimeDSS13** [comment]
changelog when
actual description when

**Comment by SonixApache** [comment]
what the fuck is that commit log

File: code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm [hunk 2]
```diff
@@ -61,7 +61,7 @@
 	..()
 	// Set up wordfilter
 	speech_filter = new
-	speech_filter.addPickReplacement("\\b(asshole|comdom|shitter|shitler|retard|dipshit|dipshit|greyshirt|nigger|security|shitcurity)",
+	speech_filter.addPickReplacement("\\b(asshole|condom|shitter|shitler|retard|dipshit|dipshit|greyshirt|nigger|security|shitcurity)",
```
Comment (SonixApache) [inline code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm]: 🤔

File: code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm [hunk 3]
```diff
@@ -164,7 +164,7 @@
 	icon_dead = null
 	response_help = "honks the"
 	speak = list("Honk!")
-	speak_emote = list("sqeaks")
+	speak_emote = list("squeaks")
```
Comment (SonixApache) [inline code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm:167]: 🤔

Comment (GermanSanta) [reply code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm]: fixing others past typos. wtf is a comdom

File: code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm (hunk 2 above)
Comment (GermanSanta) [reply code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm]: fixing others past typos. wtf is a comdom

File: code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm (hunk 3 above)
Comment (GermanSanta) [reply code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm:167]: sqeak

File: code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm (hunk 2 above)
Comment (ghost) [reply code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm]: it's not a typo, comdom is the original term and likely a bastardized portmanteau of "command" and "dominant"

**Comment by GermanSanta** [comment]
Know this is pretty self explanatory but here's my attempt at Change log and description
How to create them? The usual. 
How to kill them? Hit a lot
Honk!
🆑
rscadd: Adds psychedelic clown goblins, made from psychedelic clown outfits
rscfix: Fixes the sqeak typo???

**Comment by Exxion** [comment]
Don't believe you can have line breaks in a changelog entry

**Comment by GermanSanta** [comment]
This has been tested and works well. Everything is ready for the merge.
Just need to know if it should be squeak or sqeak

File: code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm (hunk 3 above)
Comment (SonixApache) [reply code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm:167]: this is fine

**Comment by SonixApache** [comment]
add this to the bottom of your OP
```
:cl:
 - rscadd: Added psychedelic clown goblins, made from psychedelic clown outfits
```

**Comment by GermanSanta** [comment]
All is clear

**Comment by SonixApache** [comment]
you didn't do it right

**Comment by GermanSanta** [comment]
what's wrong?

**Comment by SonixApache** [comment]
you fixed it

**Comment by GermanSanta** [comment]
everything look good from here then?

File: code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm [hunk 4]
```diff
@@ -189,3 +189,47 @@
...
+	emote_hear = list("honks")
+	maxHealth = 100
+	health = 100
+	size = 1
+	environment_smash_flags = SMASH_LIGHT_STRUCTURES
+
+	speed = 1
+	turns_per_move = 1
+
+	melee_damage_type = "BRAIN"
```
Comment (Kurfursten) [inline code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm:212]: does this actually work?

**Comment by Kurfursten** [comment]
confirm that the melee brain damage type works with testing and I will merge

**Comment by GermanSanta** [comment]
That melee damage type does not work for a fact melee_damage_type = "BRAIN". Which would be fixed by making it melee_damage_type = BRAIN. But it was like this with the original clown goblin so i just kept it as is. 

Since having a stunning, brain damaging mob would probably be controversial. I planned on creating a separate pull request as balancing issue for both this psychedelic clown goblin and the original. After the merge.

**Comment by Kurfursten** [comment]
Okay you can separate pr it as a legacy bug fix

**Pull request merged**